notebooks/04_model_select_and_optimize/aml_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>Again, I would not mention: "only supported choice". This could change and we will forget about it</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Removed.